js/scripts.js

```diff
@@ -212,3 +212,30 @@ document.querySelector('form').addEventListener('submit', (event) => {
     alert.style.visibility = 'hidden';
   }, 5000);
 });
+
+const store = () => {
+  const name = document.getElementById('form-name').value;
+  const email = document.getElementById('form-email').value;
+  const text = document.getElementById('form-text').value;
+  const formObj = {
+    name,
+    email,
+    text,
+  };
+
+  localStorage.setItem('formObj', JSON.stringify(formObj));
+};
+
+const dataObj = localStorage.getItem('formObj');
+if (dataObj) {
+  const parseObj = JSON.parse(dataObj);
+  document.getElementById('form-name').value = parseObj.name;
+  document.getElementById('form-email').value = parseObj.email;
+  document.getElementById('form-text').value = parseObj.text;
+}
```

Comment:
- [ ][OPTIONAL] This is a great job you have done so far with the storage, however, it could be nicer to add this logic when the page load like this 👇 
```
window.onload = () => {
   const dataObj = localStorage.getItem('formObj');
   if (dataObj) {
       const parseObj = JSON.parse(dataObj);
       document.getElementById('form-name').value = parseObj.name;
       document.getElementById('form-email').value = parseObj.email;
       document.getElementById('form-text').value = parseObj.text;
   }
}
```